omnibus/Gemfile

```diff
@@ -1,11 +1,12 @@
 source 'https://rubygems.org'
 
 gem 'rake'
-
+# chef-cli pulls chef >= 15.0 which is 16.1.16 at the time of failure.
+# https://github.com/chef/chef/commit/003fbc132935961e93667f11c3f45ce4914b83ac
+gem 'chef', '=15.8.23'
```

Comment:
You can probably get away with 15.11.whatever here. 16.something is where the change was made

Reply:
Right - I wanted to pin to the same version pinned in omnibus_overrides. We can upgrade the version overall once we know this change works.